src/java/org/apache/cassandra/db/partitions/PartitionUpdate.java

```diff
@@ -810,6 +814,19 @@ public void serialize(PartitionUpdate update, DataOutputPlus out, int version) t
             }
         }
 
+        public void serializeWithoutKey(PartitionUpdate update, TableMetadatas tables, DataOutputPlus out, int version) throws IOException
+        {
+            try (UnfilteredRowIterator iter = update.unfilteredIterator())
+            {
+                assert !iter.isReverseOrder();
```

Comment:
no issues for me, but is this possible?  `update.unfilteredIterator()` is explicitly not reversed

Reply:
I am fairly sure this is copy/paste. I can take another look, you’re probably right that we can remove it.